.circleci/config.yml

```diff
@@ -59,7 +59,10 @@ jobs:
       - test
 
   publish:
-    executor: node-10
+    executor:
+      name: node/default
```

Comment:
Just curious, what does node/default refer to?

Reply:
It's the pre-built docker image we want circle to use